moveit_ros/perception/depth_image_octomap_updater/src/depth_image_octomap_updater.cpp

```diff
@@ -93,6 +93,7 @@ bool DepthImageOctomapUpdater::setParams(XmlRpc::XmlRpcValue& params)
     readXmlParam(params, "shadow_threshold", &shadow_threshold_);
     readXmlParam(params, "padding_scale", &padding_scale_);
     readXmlParam(params, "padding_offset", &padding_offset_);
+    readXmlParam(params, "max_update_rate", &max_update_rate_);
```

Comment:
This parameter should be *optional* in order not to break current setups.

The default value (probably `0`?) should skip the check below and behave as before.

Reply:
done, made it skip check for <= 0

Reply:
Is that really the best default? Why not 10hz? It sounds like an unrestricted update rate is unusable.

Reply:
Unrestricted is the default until now and if users throttle/control the input on their own (best practice), a different default can affect/break their setup.
If you want it changed, we could do that in melodic and document it in the MIGRATION file, but I would just leave it as is.
The tutorial should not even mention it's optional, so nobody should run in that problem again.